src/game/components/Color.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef GAME_COMPONENTS_COLOR_HPP_
+#define GAME_COMPONENTS_COLOR_HPP_
+
+#include "ecs/Component.hpp"
+#include "raylib/core/Color.hpp"
+
+namespace game
+{
+    namespace components
+    {
+        struct Color : public ecs::Component {
```

Comment:
The structure can inherit from the class raylib::core::Color

There will be no more variable and it can be used as a raylib::core::Color.